lib/systems/examples.nix

```
@@ -221,6 +221,35 @@ rec {
     libc = "newlib";
   };
 
+  #
+  # UEFI
+  #
+
+  # See: https://github.com/rust-lang/rust/pull/56769/files#r241916113
+  i686-uefi = {
+    # config = "i686-unknown-windows-gnu";
+    config = "i686-pc-win32-coff";
```

```suggestion
    config = "i686-w64-mingw32";
```

This is the `config.sub` triple; see https://github.com/NixOS/nixpkgs/issues/226291#issuecomment-1525342950